content/en/docs/reference/access-authn-authz/admission-controllers.md

```diff
@@ -744,17 +744,35 @@ for more information.
 
 ### SecurityContextDeny {#securitycontextdeny}
 
-This admission controller will deny any Pod that attempts to set certain escalating
-[SecurityContext](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#securitycontext-v1-core)
-fields, as shown in the
-[Configure a Security Context for a Pod or Container](/docs/tasks/configure-pod-container/security-context/)
-task.
-If you don't use [Pod Security admission](/docs/concepts/security/pod-security-admission/),
-[PodSecurityPolicies](/docs/concepts/security/pod-security-policy/), nor any external enforcement mechanism,
-then you could use this admission controller to restrict the set of values a security context can take.
-
-See [Pod Security Standards](/docs/concepts/security/pod-security-standards/) for more context on restricting
-pod privileges.
+This admission controller will deny any Pod that attempts to set the following
+[SecurityContext](/docs/reference/generated/kubernetes-api/{{< param "version">}}/#securitycontext-v1-core)
+fields:
+- `Pod.Spec.SecurityContext.SupplementalGroups`
+- `Pod.Spec.SecurityContext.SELinuxOptions`
+- `Pod.Spec.SecurityContext.RunAsUser`
+- `Pod.Spec.SecurityContext.FSGroup`
+- `Pod.Spec.(Init)Containers[].SecurityContext.SELinuxOptions`
+- `Pod.Spec.(Init)Containers[].SecurityContext.RunAsUser`
+
+For more historical context on this plugin, see
+[this blog post](/blog/2022/08/23/podsecuritypolicy-the-historical-context/)
+about the PodSecurityPolicy historical context and the birth of the
+SecurityContext.
+
+{{< caution >}}
```

Comment:
I'd put this callout just under the feature state shortcode that I also recommend adding.

Reply:
I agree!